public/delete_thread.php

```diff
@@ -0,0 +1,31 @@
+<?php 
+
+	require('database.php');
+
+	session_start();
+
+	$threadId = $_POST['id'];
+
+	$authorId = $_SESSION['user_id'];
+
+	$db = new Database();
+
+	$query = "SELECT * FROM threads WHERE id = $_GET[id] AND author_id = $_SESSION[user_id]";
+
+	$result = $db->query($query);
+
+	if ($result === false) {
+		header("Location: http://rdripley.com/forum");
+	}
+
+	$threadsQuery = "DELETE FROM threads WHERE thread_id= $threadId";
+
+	$postsQuery = "DELETE FROM posts WHERE thread_id= $threadId AND author_id= $authorId";
```

Comment:
The `AND author_id` part of this is problematic. It will make it only delete posts by the thread creator instead of all posts from the thread.